src/newspack-ui/style.scss

```diff
@@ -3,15 +3,14 @@
 @use "scss/modals";
 
 .newspack-ui {
-
 	// Need better spot for this
 	&__color-text-gray {
 		color: var(--newspack-ui-color-neutral-60);
 	}
 
 	// Overrides for theme styles -- to move to theme?
-	h2,
-	h3 {
+	h2:not([class*="font-size"]),
+	h3:not([class*="font-size"]) {
```

Comment:
It looks like these styles are affecting header Registration Block: 

![CleanShot 2024-10-07 at 11 53 01](https://github.com/user-attachments/assets/e08fd172-93e7-45d6-8ef6-6655dbdc16a6)

Compared to the mockup:

![CleanShot 2024-10-07 at 11 50 08](https://github.com/user-attachments/assets/20d841cf-74f3-4e36-b5ce-c5e9d8e3281a)

It looks like the title is set to the large font size [here](https://github.com/Automattic/newspack-plugin/pull/3199/files#diff-073e276f3eb477174d7918af604a017b086f70e9e9ffc0d305c57158c23a24aeR98) so I think it's just a specificity thing.

[The theme uses classes with `font-size`](https://github.com/Automattic/newspack-theme/blob/0ccc70d6e686575b04ee2e24555b96af0a34dca2/newspack-theme/sass/blocks/_blocks.scss#L1420), but WordPress adds `!important` to those as part of Gutenberg. [Newspack UI just uses `font`](https://github.com/Automattic/newspack-plugin/blob/epic/ras-acc/src/newspack-ui/scss/elements/_typography.scss#L51C4-L51C12) -- I think it may be safe to remove the `:not` unless this is meant to protect against something else?

Reply:
These styles were added because the less-specific selectors were overriding font sizes set in the editor:

| Editor | `epic/ras-acc` | this branch|  
|--------------|-----------|---|
| <img width="845" alt="image" src="https://github.com/user-attachments/assets/04b67dce-6d2a-40f7-bc86-663fd3563b1d"> | <img width="1043" alt="image" src="https://github.com/user-attachments/assets/fd98c3b2-1963-4684-be64-55001770fd38"> | <img width="1045" alt="image" src="https://github.com/user-attachments/assets/8b5e032f-741c-4611-8077-e5653846d94b"> |

I've added `!important`s in 96afca6e1a342dc2976cdd0c7caad8f5acadb755, would that be ok?